install/update_91_92.php

```diff
@@ -866,6 +866,132 @@ function update91to92() {
+               `entities_id` int(11) NOT NULL DEFAULT '0',
+               `is_recursive` tinyint(1) NOT NULL DEFAULT '0',
+               `manufacturers_id` int(11) NOT NULL DEFAULT '0',
+               `voltage` int(11) DEFAULT NULL,
+               `devicesimcardtypes_id` int(11) NOT NULL DEFAULT '0',
+               `date_mod` datetime DEFAULT NULL,
+               `date_creation` datetime DEFAULT NULL,
+               `allowvoip` tinyint(1) NOT NULL DEFAULT '0',
+               PRIMARY KEY (`id`),
+               KEY `designation` (`designation`),
+               KEY `entities_id` (`entities_id`),
+               KEY `is_recursive` (`is_recursive`),
+               KEY `devicesimcardtypes_id` (`devicesimcardtypes_id`),
+               KEY `date_mod` (`date_mod`),
+               KEY `date_creation` (`date_creation`),
+               KEY `manufacturers_id` (`manufacturers_id`)
+            ) ENGINE=MyISAM  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci";
+        $DB->queryOrDie($query, "9.2 add table glpi_devicesimcards");
+      }
+
+      if (!TableExists('glpi_items_devicesimcards')) {
+      $query = "CREATE TABLE IF NOT EXISTS `glpi_items_devicesimcards` (
+                  `id` int(11) NOT NULL AUTO_INCREMENT,
+                  `items_id` int(11) NOT NULL DEFAULT '0' COMMENT 'RELATION to various table, according to itemtype (id)',
+                  `itemtype` varchar(100) COLLATE utf8_unicode_ci NOT NULL,
+                  `devicesimcards_id` int(11) NOT NULL DEFAULT '0',
+                  `is_deleted` tinyint(1) NOT NULL DEFAULT '0',
+                  `is_dynamic` tinyint(1) NOT NULL DEFAULT '0',
+                  `entities_id` int(11) NOT NULL DEFAULT '0',
+                  `serial` varchar(255) NULL DEFAULT NULL,
+                  `otherserial` varchar(255) NULL DEFAULT NULL,
+                  `states_id` int(11) NOT NULL DEFAULT '0',
+                  `locations_id` int(11) NOT NULL DEFAULT '0',
+                  `lines_id` int(11) NOT NULL DEFAULT '0',
+                  `pin` varchar(255) NOT NULL DEFAULT '',
+                  `pin2` varchar(255) NOT NULL DEFAULT '',
+                  `puk` varchar(255) NOT NULL DEFAULT '',
+                  `puk2` varchar(255) NOT NULL DEFAULT '',
+                  PRIMARY KEY (`id`),
+                  KEY `item` (`itemtype`,`items_id`),
+                  KEY `devicesimcards_id` (`devicesimcards_id`),
+                  KEY `is_deleted` (`is_deleted`),
+                  KEY `is_dynamic` (`is_dynamic`),
+                  KEY `entities_id` (`entities_id`),
+                  KEY `serial` (`serial`),
+                  KEY `otherserial` (`otherserial`),
+                  KEY `states_id` (`states_id`),
+                  KEY `locations_id` (`locations_id`),
+                  KEY `lines_id` (`lines_id`)
+                ) ENGINE=MyISAM  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci";
+      $DB->queryOrDie($query, "9.2 add table glpi_items_devicesimcards");
+   }
+
```

Comment:
After browsing the current table, it seems there is no display preference for device* by default. I suggest we keep this behavior for simcards too